src/interfaces/banner.ts

```diff
@@ -88,6 +88,10 @@ export type Action =
       label: 'Check'
       actionName: 'confirm-temp-seed'
     }
+  | {
+      label: 'Open'
+      actionName: 'open-first-cashback-confetti-modal'
```

Comment:
Ideally, we want to skip the UI specifics (confetti), since ethey're not relevant on ambire-common (business logic) level, I'd suggest changing this to: 'open-first-cashback-modal' only.